beacon_node/src/config.rs

```diff
@@ -143,6 +143,24 @@ pub fn get_config<E: EthSpec>(
         client_config.http_metrics.allow_origin = Some(allow_origin.to_string());
     }
 
+    /*
+     * Explorer metrics
+     */
+    if cli_args.is_present("explorer-metrics") {
+        let explorer_endpoint = cli_args
+            .value_of("explorer-address")
+            .expect("guaranteed by clap")
```

Comment:
I would just return an error with some useful text here, it'll be nicer than a `panic: guaranteed by clap` if we happen to mess up the clap config.